### PR TITLE
handle data uris

### PR DESCRIPTION
**Pull request by chrisplim**
Handle uploading of data_uris correctly, as a normal upload instead of a local file

### CODE REVIEW

File: lib/cloudex.ex [hunk 1]
```diff
@@ -54,12 +54,12 @@ defmodule Cloudex do
   @spec upload_list_with_options(list(map), map) :: upload_result
   def upload_list_with_options(list, default_options \\ %{}) do
     list
-    |> Enum.map(fn %{url: url} = item ->
-      item_specific_options = Map.drop(item, :url)
+    |> Enum.map(fn %{image_resource: image_resource} = item ->
```
Comment (camilohollanda) [inline lib/cloudex.ex]: @chrisplim I prefer to keep url, it is more precise wording

Comment (chrisplim) [reply lib/cloudex.ex]: it's more accurate to have image_resource, since this is not limited to urls. it can be files, uris or urls (subset of uri)

Comment (camilohollanda) [reply lib/cloudex.ex]: but then what if it is a video?

Comment (chrisplim) [reply lib/cloudex.ex]: true, i'll just label it resource

**Review by camilohollanda — APPROVED** [review]

**Pull request merged**